### DISCUSSION TITLE
Help with optimizing config

### DISCUSSION THREAD

**Discussion by Kali123411** [swar/Swar-Chia-Plot-Manager#504] [Q&A]
So I just added two new SSD to my rig today, two are NVME and one is SATA3. Looking at the plot manager they appear to be plotting about equal speed which is weird because I assumed the SATA3 SSD would be a lot slower. I was hoping someone could comment on my config and see where I have areas for improvement. My CPU and RAM are not even maxing out with my current config.
![config](https://user-images.githubusercontent.com/64617127/118549709-1471cd80-b719-11eb-88fd-3ebbef75d3b0.JPG)
![phase1kickoff](https://user-images.githubusercontent.com/64617127/118549724-18055480-b719-11eb-9947-f6466c7ebaf9.JPG)
![cpu](https://user-images.githubusercontent.com/64617127/118549728-19cf1800-b719-11eb-8e93-216fb7db2dfe.JPG)
![memory](https://user-images.githubusercontent.com/64617127/118549730-1b004500-b719-11eb-9a0c-42f7253f583d.JPG)

**Comment by Icabaad** [comment]
Just a quick readover.... You have set to many threads for your CPU. This is threads per plot not how many are on your CPU. SHould be more like 2. Is your max concurrent to high? You are saying run upto 14 plots per HDD. 2TBs are normally 6-7 plots in parallel. Not all m2 drives are made the same. Their writes per second can drop quite a bit after a little time dramatically depending on quality of drive. Temperature can have an effect as well.

**Reply by Kali123411** [reply]
Thanks, I realized after running this config that I was over saturating my CPU and also RAM. I am installing a new CPU hopefully today Xeon E5-2683v3. After this new CPU gets installed I will probably be limited by my 32GB of RAM. I am looking at possibly upgrading to 64GB RAM if I can find a deal.

**Comment by Lyushen** [comment]
How d'fk you processing 6 on phase one PER TASK? I am with my 10 cores 10850k only can handle 3 or 5 threads per phase1 in general (from 3 tasks),  and still debating with myself what is better, 3 or 5. The pase 1 is super CPU heavy. Try to lower it to 1 or max 2 per task
